diktat-gradle-plugin/src/main/kotlin/org/cqfn/diktat/plugin/gradle/DiktatJavaExecTaskBase.kt

```diff
@@ -25,7 +25,7 @@ import javax.inject.Inject
  *
  * Note: class being `open` is required for gradle to create a task.
  */
-open class DiktatJavaExecTaskBase @Inject constructor(
+class DiktatJavaExecTaskBase @Inject constructor(
```

Comment:
Did you see note in KDoc? It's required for this class to be open

Reply:
ahahahaha